README.md

```diff
@@ -38,9 +38,6 @@ APIs are updated frequently, we release a **new major version at the end of
 each month** with a full changelog. For more details and background see [SDK Update
 Practices](https://github.com/Azure/azure-sdk-for-go/wiki/SDK-Update-Practices).
 
-To more reliably manage dependencies like the Azure SDK in your applications we
```

Comment:
There is more mention of `dep` below that should be removed.  And `gopkg.toml` which is the `dep` dependency data file.